tests/basic/pure_test.go

```diff
@@ -2810,3 +2813,71 @@ var _ = Describe("{VolAttachSameFAPxRestart}", func() {
 		defer EndTorpedoTest()
 	})
 })
+
+/*
+This test deploys app with FBDA volume having storageClass with pure_nfs_endpoint parameter.
+It validates that FBDA volume gets consumed over IP mentioned in `pure_nfs_endpoint` parameter of storageClass.
+*/
+var _ = Describe("{FBDAMultiTenancyBasicTest}", func() {
+	var contexts []*scheduler.Context
+	var testName string
+	var customConfigAppName string
+
+	testName = "fbda-multitenancy"
+	JustBeforeEach(func() {
+		StartTorpedoTest("FBDAMultiTenancyBasicTest", "Validate FBDA vols get consumed over IP mentioned in `pure_nfs_endpoint` parameter of storageClass", nil, 0)
+		Step("setup credential necessary for cloudsnap", createCloudsnapCredential)
+
+		logrus.Infof("Using CustomAppConfig: %+v", Inst().CustomAppConfig)
+		contexts = ScheduleApplications(testName)
+		ValidateApplicationsPureSDK(contexts)
+		// Skip the test if we don't find any of our apps
+		for appName := range Inst().CustomAppConfig {
+			found := false
+			for _, ctx := range contexts {
+				if ctx.App.Key == appName {
+					found = true
+					customConfigAppName = appName
+					break
+				}
+			}
+			if !found {
+				logrus.Warnf("App %v not found in %d contexts, skipping test", appName, len(contexts))
+				Skip(fmt.Sprintf("app %v not found", appName))
+			}
+		}
```

Comment:
Can we do this before scheduling the apps?

Reply:
done